src/Norms.jl

```diff
@@ -211,7 +210,7 @@ function dfly(N1::Type{TotalVariation}, N2::Type{L1}, D::PwMap)
     end
     
     dist = max_distortion(D)
-    lam = expansivity(D)
+    lam = max_expansivity(D)
```

Comment:
Good point. What about `max_inverse_expansivity`? This would make sense as both "the inverse of the expansivity" and "the expansivity of the inverse function of $f$". Or if there you know a better way to call the inverse of the derivative let me know.

Reply:
I think `max_inverse_derivative` is probably the best option.
Looking online for a definition of expansivity it does not correspond to our meaning.
So, for the sake of simplicity, I think `max_inverse_derivative` is probably best...
What do you think?

Reply:
`max_inverse_derivative` (and `inverse_derivative` for the function) sounds good. :+1: I'll fix the PR.